### DISCUSSION TITLE
Joystick gremlin not opening ever since i downloaded it

### DISCUSSION THREAD

**Discussion by KatKiing** [WhiteMagic/JoystickGremlin#582] [Q&A]
![image](https://github.com/user-attachments/assets/b6512056-fb19-47ec-99f2-c351ec5caa04)
i have hidhide and vJoy, those both open but this one has never worked for me.

**Comment by WhiteMagic** [comment]
This can be a myriad of things from the vJoy setup, to what you blocked and whitelisted in hidhide, to odd quirks with vJoy. For any reasonable help, info about the setups of the vjoy and hidhide is needed. You can also drop by he discord hotas (https://discord.gg/szqaJE7) for more interactive help in the joystick gremlin channel.
